### PR TITLE
[Ceph-Mgr][Tier2]Verify mgr balancer

### PR DESCRIPTION
**Pull request by MohitBis**
[Ceph-Mgr][Tier2]Verify mgr balancer

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ROODM4/

File: cli/ceph/config_key.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        self.base_cmd = f"{base_cmd} config-key"
+
+    def set(self, key, value):
+        """set config key to value
+        Args:
+            key (str): config key
+            value (str): config key value
+        """
+        cmd = f"{self.base_cmd} set {key} {value}"
+        return self.execute(sudo=True, long_running=True, cmd=cmd)
```
Comment (vamahaja) [inline cli/ceph/config_key.py:19]: missing `check_ec` ?

File: suites/pacific/cephmgr/tier-2_verify_throttling_mgr_balancer.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/pacific/cephmgr/tier-2_verify_throttling_mgr_balancer.yaml:1]: Does new test suite file required here ? Can't be existing test suite used?

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: cli/ceph/balancer.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+class Balancer(Cli):
+    """This module provides CLI interface to manage the balancer module."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Balancer, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} balancer"
+
+    def status(self):
+        """Turn On balancer module."""
```
Comment (pranavprakash20) [inline cli/ceph/balancer.py:13]: To confirm, the status method is to return the status of the balancer? Or is it to turn on the module? If not, please update the method method info here.

File: cli/ceph/config_key.py (hunk 1 above)
Comment (MohitBis) [reply cli/ceph/config_key.py]: Thanks @vamahaja 
Updated

File: suites/pacific/cephmgr/tier-2_verify_throttling_mgr_balancer.yaml (hunk 2 above)
Comment (MohitBis) [reply suites/pacific/cephmgr/tier-2_verify_throttling_mgr_balancer.yaml]: Thanks @vamahaja
Updated in existing suite

File: cli/ceph/balancer.py (hunk 3 above)
Comment (MohitBis) [reply cli/ceph/balancer.py]: Thanks @pranavprakash20 
Updated

**Comment by MohitBis** [comment]
Latest Log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2F600E/

**Review by pranavprakash20 — APPROVED** [review]
LGTM

File: tests/mgr/test_cephmgr_throttling.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            polarion-id: CEPH-83573248
+            module: test_cephmgr_throttling
+            config:
+                operation: enable
+                module: balancer
+                key: target_max_misplaced_ratio
+                value: .07
+    """
+    node = ceph_cluster.get_nodes(role="mon")[0]
+    action = kw.get("config").get("operation")
```
Comment (vamahaja) [inline tests/mgr/test_cephmgr_throttling.py:25]: Add validations for these config parameters

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: tests/mgr/test_cephmgr_throttling.py (hunk 4 above)
Comment (MohitBis) [reply tests/mgr/test_cephmgr_throttling.py:25]: Thanks @vamahaja 
Updated

**Review by vamahaja — APPROVED** [review]

**Pull request merged**